### DISCUSSION TITLE
Can we use the Trigger Build Task to trigger from a Tag instead of a Branch?

### DISCUSSION THREAD

**Discussion by SA-Egress** [maikvandergaag/msft-extensions#531] [Q&A] [answered]
https://marketplace.visualstudio.com/items?itemName=maikvandergaag.maikvandergaag-trigger-pipeline

Can we use this extension/Trigger Build Task to trigger from a Tag instead of a Branch?

**Comment by maikvandergaag** [comment, marked as answer]
no, not yet that should be implemented when required. we could add a feature request for that.

**Comment by SA-Egress** [comment]
That would be handy!
